src/Cms/Resource/incremental/mysql/0073_cms_file_index_optimization.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `cms_file`
+DROP INDEX `object`,
```

Comment:
Under `object`, object+objectId (2 field index) was added, and it wasn't a good idea performance wise. Separate indexes work better.